run.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 python main.py /mnt/efs/Trees_processed/ -j=2 --lr=0.0001 -b=16 \
```

Comment:
Can you use longer notation for your variable like J lr B, etc?

That way users can more easily use it without have to look up documentation

Reply:
Those short options actually have corresponding long options. For example 
`-j = --num-workers`